apps/examples/testcase/le_tc/kernel/tc_libc_misc.c

```diff
@@ -347,6 +349,72 @@ static void tc_libc_misc_vdbg(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+ * @fn                  :tc_libc_misc_dumpbuffer
+ * @brief               :Do a pretty buffer dump
+ * @scenario            :Do a pretty buffer dump
+ * @API's covered       :lib_dumpbuffer
+ * @Preconditions       :None
+ * @Postconditions      :None
+ * @Return              :void
+ */
+static void tc_libc_misc_dumpbuffer(void)
+{
+	const char *msg = "tc_libc_misc_dumpbuffer";
+	unsigned char buffer[] = { 'S', 'A', 'M', 'S', 'U', 'N', 'G', '-', 'T', 'i', 'z', 'e', 'n', 'R', 'T' };
+	unsigned char *buf = NULL;
+	int idx;
+
+	/**
+	 * As lib_dumpbuffer returns void, there in no way to check the success or failure case
+	 * we need to manually check the dump output
+	 * Case-1: output: 53414d53554e472d54697a656e5254 "SAMSUNG-TizenRT"	only
```

Comment:
I guess you understood what I said.
I didn't mean adding " " and only to comments so please remove these